src/scales/scale.linear.js

```diff
@@ -12,6 +12,31 @@ module.exports = function(Chart) {
 	};
 
 	var LinearScale = Chart.LinearScaleBase.extend({
+
+		scaleLimitSnapshot: {
+			max: null,
+			min: null
+		},
+
+		/**
+		 * A util function to create snapshot of axis limits {max, min}
+		 * @Method chart.takeScaleLimitSnapshot
+		 * @param Integer {min} chart mininmum
```

Comment:
The JSDoc comments aren't quite correct syntax wise. The data type is inside `{}`. Further, these aren't just integers since the scale could use floating point.

`* param {Number} min - chart minimum